docroot/modules/custom/civictheme_migrate/assets/migration_schemas/menu_link_content_civictheme_primary.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
please provide 3 schemas in total - 1 for each menu because we use 3 migrations and they all require validations

Reply:
@AlexSkrypnyk If you check I have used same bundle key in migration file so they can use same schema.
Default bundle key is not required in these migration so used in different way.

Reply:
this is not for migrations but for validations

Reply:
@AlexSkrypnyk All 3 migration will use the same schema for the validation.